docs/getting_started/examples.rst

```diff
@@ -0,0 +1,67 @@
+
+Examples and Resources 
+========================
+
+The earlier sections of have covered the basics of using Modin to speed up your pandas workflow. Here you can find additional resource to learn about Modin. To learn more about advanced usage for Modin, please refer to :doc:`this section </advanced_usage/index>`.
+
+Basic Usage Examples
+---------------------
+
+The following notebooks demonstrate how Modin can be used for scalable data science: 
+
+- Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
```

Comment:
Getting a "Page Not Found" here as well as in a few of the exercises in the tutorial section.

Reply:
This will only work after this PR is merged in.